src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -71,6 +71,7 @@
     "d:transactions/breakdowns.span_ops.ops.browser@millisecond": PREFIX + 122,
     "d:transactions/breakdowns.span_ops.ops.resource@millisecond": PREFIX + 123,
     "d:transactions/breakdowns.span_ops.ops.ui@millisecond": PREFIX + 124,
+    "c:transactions/count_per_root_project@none": PREFIX + 125,
```

Comment:
For reference: When adding strings here, we must make sure that they don't already exist in the db. This is improbable for new MRIs, but better safe than sorry. 

https://github.com/getsentry/sentry/blob/9a0ae70e29bbe4eb4e39db766da5f342ca37cb06/src/sentry/sentry_metrics/indexer/strings.py#L18-L19

```sql
select * from sentry_perfstringindexer where string = 'c:transactions/count_per_root_project@none';
select * from sentry_stringindexer where string = 'c:transactions/count_per_root_project@none';
```

Both queries yield 0 results, so it seems we are good.

Reply:
@jjbayer good catch! PR title has been updated.